dex-load/src/main/resources/reinstallExtension.sh

```diff
@@ -0,0 +1,7 @@
+systemctl stop waves-devnet.service || true
+dpkg -r waves-dex-extension-devnet || true
+dpkg -P waves-dex-extension-devnet || true
+dpkg -i /home/buildagent-matcher/waves-dex-extension-devnet*.deb
```

Comment:
I didn't see cleanup after previous deployments in build steps. So this line probably try to install multiple versions of waves-dex-extension-devnet